<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>toolbar_renderenhancegroup</class>
 <widget class="QWidget" name="toolbar_renderenhancegroup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>860</width>
    <height>105</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Render Enhance Group</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>6</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>6</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="verticalSpacing">
    <number>3</number>
   </property>
   <item row="0" column="0" colspan="2">
    <widget class="QCheckBox" name="checkBox_edgeAwareBlur">
     <property name="text">
      <string>Smooth render</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_resolution">
     <property name="text">
      <string>Res.</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="comboBox_edgeAwareResolution"/>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="label_radius">
     <property name="text">
      <string>Radius</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QSlider" name="slider_edgeAwareRadius">
     <property name="minimumSize">
      <size>
       <width>80</width>
       <height>0</height>
      </size>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QSpinBox" name="spinBox_edgeAwareRadius"/>
   </item>
   <item row="1" column="2">
    <widget class="QLabel" name="label_edgeSensibility">
     <property name="text">
      <string>Edge sens.</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QSlider" name="slider_edgeAwareDepth">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <widget class="QSpinBox" name="spinBox_edgeAwareDepth"/>
   </item>
   <item row="2" column="2">
    <widget class="QLabel" name="label_mix">
     <property name="text">
      <string>Mix</string>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QSlider" name="slider_edgeAwareBlend">
     <property name="minimumSize">
      <size>
       <width>80</width>
       <height>0</height>
      </size>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="4">
    <widget class="QSpinBox" name="spinBox_edgeAwareBlend"/>
   </item>
   <item row="0" column="5">
    <widget class="QCheckBox" name="checkBox_depthLining">
     <property name="text">
      <string>Contours</string>
     </property>
    </widget>
   </item>
   <item row="0" column="6">
    <widget class="QCheckBox" name="checkBox_depthLiningStrongMode">
     <property name="text">
      <string>Intense mode</string>
     </property>
    </widget>
   </item>
   <item row="1" column="5">
    <widget class="QLabel" name="label_depthLiningSensitivity">
     <property name="text">
      <string>Sensitivity</string>
     </property>
    </widget>
   </item>
   <item row="1" column="6">
    <widget class="QSlider" name="slider_depthLiningSensitivity">
     <property name="minimumSize">
      <size>
       <width>80</width>
       <height>0</height>
      </size>
     </property>
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="value">
      <number>60</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="7">
    <widget class="QSpinBox" name="spinBox_depthLiningSensitivity">
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="value">
      <number>60</number>
     </property>
    </widget>
   </item>
   <item row="2" column="5">
    <widget class="QLabel" name="label_depthLiningStrength">
     <property name="text">
      <string>Strength</string>
     </property>
    </widget>
   </item>
   <item row="2" column="6">
    <widget class="QSlider" name="slider_depthLiningStrength">
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="value">
      <number>45</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="7">
    <widget class="QSpinBox" name="spinBox_depthLiningStrength">
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="value">
      <number>45</number>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
